hack/newTargetGen/scaffolding/config/301-newtarget.yaml

```diff
@@ -0,0 +1,123 @@
+# Copyright 2021 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: {{.Name}}s.targets.triggermesh.io
+  labels:
+    knative.dev/crd-install: 'true'
+    duck.knative.dev/addressable: 'true'
+    triggermesh.io/crd-install: 'true'
+  annotations:
+    registry.triggermesh.io/acceptedEventTypes: |
+      [
+        { "type": "*" }
+      ]
+    registry.knative.dev/eventTypes: |
+      [
+        { "type": "io.triggermesh.{{.Name}}.response" }
+      ]
+spec:
+  group: targets.triggermesh.io
+  names:
+    kind: {{.UppercaseName}}
+    plural: {{.Name}}s
+    categories:
+    - all
+    - knative
+    - eventing
+    - targets
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        type: object
+        description: TriggerMesh event target for Alibaba Object Storage Service.
```

Comment:
Nope

I guess you could set this to "TriggerMesh event target for {{.Name}}." which good enough to bootstrap the CRD.